src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/cname/CnameContentQuery.kt

```diff
@@ -0,0 +1,25 @@
+package no.skatteetaten.aurora.gobo.graphql.cname
+
+import com.expediagroup.graphql.generator.annotations.GraphQLIgnore
+import com.expediagroup.graphql.server.operations.Query
+import no.skatteetaten.aurora.gobo.integration.spotless.SpotlessCnameService
+import org.springframework.stereotype.Component
+
+data class CnameContent(
```

Comment:
Er det mulig å gjenbruke CnameInfo og CnameEntry fra CnameInfoQuery.kt? Og sende inn parameter for hvilken type Cname man ønsker (OnPrem (Gavel), Azure (Spotless), etc.).

Reply:
Det blir ikke helt optimalt, vi ender opp med ganske mange felt som blir null, som kan gjøre det vanskeligere å bruke APIet.

Reply:
Jeg ser at du har gjort endringen @furode . Jeg er usikker på hva vi tjener på det @stromland . Jeg synes det ble forvirrende i playground og litt rar graf av dette. 

Hva hvis vi heller prøver å gruppere det under samme sted på grafen. De har jo ganske ulike felt
```
cname {
   azure{
   }
   onPrem{}
   }
   ```

Reply:
Jeg har endret det slik at det blir likt modellen til @kris456 og toleranse for at en av de kan få feil men fremdeles få resultatet for den andre.